python/dlisio/lis/load.py

```diff
@@ -101,6 +98,7 @@ class FileIndexer:
     def __init__(self, path, error_handler):
         self.error_handler = error_handler
         self.path = path
+        self.complete = False
```

Comment:
The attribute `self.file` on line 108 is no longer used, except within `index_logical_file`. There is no need to cache it on the class